upgrade/helpers/docker.py

```diff
@@ -91,7 +91,7 @@ def refresh_subscriptions_on_docker_clients(container_ids):
         docker_execute_command(container_ids, 'yum clean all', quiet=True)
 
 
-def docker_execute_command(container_id, command, quiet=False, async=False):
+def docker_execute_command(container_id, command, quiet=True, async=False):
```

Comment:
Why is this change required ?

Reply:
So that if any one of the client step , like refresh subscription, add subscrition or client registration and uograde fails for any 1 client , currently the whole job fail's

Reply:
@san7ket , I think there was a purpose why I did that. Lets get this merged and will see later if it breaks something.